tests/functional/spec/timeout_modal.spec.js

```diff
@@ -0,0 +1,11 @@
+describe("Timeout Modal", () => {
```

Comment:
I think we should cover the happy paths, such as:
- clicking `Continue` extends the session.
- ensure it works on post submission pages

Reply:
Added tests of `Continue` button, modal on post submission page and extending session when the page receives focus (by returning from another tab). Thanks!